### ISSUE TITLE
HDL File Icon Request

### ISSUE THREAD

**Issue by Remillard** [ihodev/a-file-icon#142]
I would like to request a distinct icon for HDL files.  I think something like: https://www.iconfinder.com/icons/883750/chip_circuit_computer_cpu_microchip_processor_semiconductor_icon would be appropriate, however this exact icon appears to be licensed (they're asking a buck for it).  Something like this made yourself however would be entirely appropriate.  If I had my druthers on color, default of blue might be good but I'm not an aesthetician and I don't know how various colors translate into all the various color schemes of light and dark.  I just like blue :-)

By way of explanation, HDL stands for Hardware Definition Language, and is generally used for creating and modeling complex digital hardware systems.  More typically it is the source used to create FPGA and ASIC designs.

There are two main languages and then adding support for a third would be a good idea.
1) VHDL
Typical file extensions are: *.vhd *.vhdl *.vho *.vht
Source scope: source.vhdl

2) Verilog
Typical file extensions are: *.v (and there may be others -- I'm not as familiar with Verilog as I am with VHDL, however in my Verilog *.tmLanguage file, they're only looking for *.v and *.V)
Source scope: source.verilog

3) SystemVerilog
Typical file extensions are: *.sv
Source scope: source.systemverilog

I think this is all you asked for on the Discord channel, but if there's other information you need please don't hesitate to ask.

**Comment by ihodev** [comment]
Hello @Remillard ,

I always trying to use icons from [Atom File Icons](https://github.com/file-icons/atom) package, because many people like them. 

It uses 3 different icons:

| VHDL        | Verilog           | SystemVerilog  |
|:-------------:|:-------------:|:-----:|
| ![image](https://user-images.githubusercontent.com/9801624/31159648-2f98607c-a8d3-11e7-8674-0e7b77a7beb0.png) | ![image](https://user-images.githubusercontent.com/9801624/31159666-56f014e4-a8d3-11e7-9067-b5d6bc0c30d2.png) | ![image](https://user-images.githubusercontent.com/9801624/31159702-926c3962-a8d3-11e7-8cd8-5a80a37e1361.png) |

I think there is no need in 3 "different" icons, and **VHDL** icon should fit our needs. What do you think?

> I just like blue :-)

No problem, it will be blue by default :smile: 

> If there's other information you need please don't hesitate to ask.

More than enough :wink: 

Thanks!

**Comment by Remillard** [comment]
I agree and do not believe there is much need to distinguish between the three languages.  And of the three I believe the VHDL marked one is the most 'accurate'.  Showing an 8 pin SOIC package for an HDL language is kind of weird (we are usually working on devices in the 10's to 100's of I/O pins, not counting power and ground and all of that.

Thanks for looking into this.  I got your note on Discord too and appreciate the work.
